R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -26,7 +26,7 @@ sc <- sparkR.init()
 sqlContext <- sparkRSQL.init(sc)
 
 test_that("glm and predict", {
-  training <- createDataFrame(sqlContext, iris)
+  training <- suppressWarnings(createDataFrame(sqlContext, iris))
```

Comment:
there are a number of suppressWarnings in this PR. Instead of adding notes for them in this PR, I'd like to add a note in SPARK-11976 that cleaning these warnings is part of the job. Is it OK?